### ISSUE TITLE
AttributeError: 'Dice' object has no attribute 'roll'

### ISSUE THREAD

**Issue by UtilityHotbar** [Bernardo-MG/dice-notation-python#26]
Following the example use case I have entered:

```python
from dice_notation.parser import DiceParser
parser = DiceParser()
dice = parser.parse('1d6+1')
dice.roll()
```

At which point it throws an error:

`AttributeError: 'Dice' object has no attribute 'roll'`

It appears that `Dice` instead of `RollableDice` objects are being generated.

**Comment by Bernardo-MG** [comment]
Thanks, I'll take a look as soon as I have some time.

**Comment by Bernardo-MG** [comment]
I've just released v1.2.1. Can you check it?

**Comment by UtilityHotbar** [comment]
Hey, I think it works now. Thanks!
